src/go/k8s/tests/e2e/update-image-and-node-port/00-current-image.yaml

```diff
@@ -29,11 +29,11 @@ spec:
     underReplicatedPartitionThreshold: 0.00
   resources:
     requests:
-      cpu: 100m
-      memory: 100Mi
+      cpu: 1
+      memory: 1Gi
```

Comment:
Is there enough resources in the test infra to allocate this much?

Reply:
We are running 8 worker and 1 control nodes in kind cluster. We have parallelization of 2 https://github.com/redpanda-data/redpanda/blob/82adab848f0886ae03e5d1c086f1093c79b78db1/src/go/k8s/kuttl-test.yaml#L29 so there could be at most 6 Redpanda brokers so 6CPU.

The following images shows the runner:  
![Screenshot 2023-03-01 at 22 55 42](https://user-images.githubusercontent.com/22834495/222273836-30bd449a-9254-4891-9fc4-89d9e7da90c0.png)
```
aws:instance-type=m5d.12xlarge
```

According to AWS documentation:
| Instance size | CPU | Memory | Instance Storage |
|---------------|-----|--------|------------------|
| m5d.12xlarge  | 48  | 192    | 2 x 900 NVMe SSD |

REF
https://aws.amazon.com/ec2/instance-types/m5/